modules/sample/src/test/ts/utils/Utils.ts

```diff
@@ -0,0 +1,42 @@
+import Promise from '@ephox/wrap-promise-polyfill';
+
+const post = (url: string, data: Record<string, any>): Promise<void> => {
+  return new Promise((onSuccess, onFailure) => {
+    const request = new XMLHttpRequest();
+    request.onreadystatechange = function () {
+      if (request.readyState === 4) {
+        if (request.status == 200) {
+          onSuccess();
+        } else {
+          onFailure();
+        }
+      }
+    };
+
+    request.onerror = function (err) {
+      debugger;
```

Comment:
Just wondering if we still want this debugger or should it be commented out?

Reply:
Not sure to be honest... it was there in the `IFramePassTest.ts` where I copied this from, so not sure why it was added originally.

Reply:
Going to leave it as is, as it's not going to harm anything in this case since it's not in the shipped code, it's just in the sample project which is only used by bedrock for its own tests.